bigquery-antipattern-recognition/src/test/java/com/google/zetasql/toolkit/antipattern/parser/IdentifyCTEsEvalMultipleTimesTest.java

```diff
@@ -73,7 +73,7 @@ public void withThreeCTEsTest() {
   // subquery
   @Test
   public void withOneCTETest() {
-    String expected = "CTE with multiple references: alias a defined at line 2 is referenced 2 times.";
+    String expected = "";
```

Comment:
why are we no longer detecting this pattern in this example? The below query still seems a example case for this anti pattern

Reply:
This is fixed in the latest commit please check it out